_includes/pbc/all/install-features/202307.0/install-the-service-points-customer-account-management-feature.md

```diff
@@ -152,6 +160,25 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 }
 ```
 
+### 4) Set up FE part
+
+#### 4.1) Add `main-overlay` molecule to the `page-layout-main` template.
```

Comment:
If you remove the `### 4) Set up FE part` header, then accept the following suggestions:
```suggestion
1) To the `page-layout-main` template, add the `main-overlay` molecule:
```